### PR TITLE
{bio}[foss/2023b] Mustache v1.3.3, hic-straw v1.3.1, cooler v0.10.2, multiprocess v0.70.16

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/442
requires dill-0.3.8 from:
- ~~https://github.com/easybuilders/easybuild-easyconfigs/pull/21169~~
- https://github.com/easybuilders/easybuild-easyconfigs/pull/21808

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
I've opened a separate PR for `dill`, so once that's merged this one should be ready to go:
* https://github.com/easybuilders/easybuild-easyconfigs/pull/21808

File: easybuild/easyconfigs/m/mustache-hic/mustache-hic-1.3.3-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonPackage'
+
+name = 'mustache-hic'
```
Comment (boegel) [inline easybuild/easyconfigs/m/mustache-hic/mustache-hic-1.3.3-foss-2023b.eb:3]: @pavelToman Why not just use `Mustache` here as software name?

`mustache-hic` is just what's it's called on PyPI, but that's a workaround for a naming clash with https://pypi.org/project/mustache, which seems like a dead project...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3103.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/7fd9b2691650de8bce4ed0defde7dea1 for a full test report.

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3103.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/e74a6851fec8be1fef678aee0f977ba9 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**